config/clusters/nasa-veda/common.values.yaml

```
@@ -96,6 +96,17 @@ basehub:
                   slug: pangeo
                   kubespawner_override:
                     image: public.ecr.aws/nasa-veda/nasa-veda-singleuser:5068290376e8c3151d97a36ae6485bb7ff79650b94aecc93ffb2ea1b42d76460
+                    init_containers:
+                      - name: nasa-veda-singleuser-init
+                        image: public.ecr.aws/nasa-veda/nasa-veda-singleuser-init:1fd99797107550a50d58ce7e5dd4042e37a065ac2f88576bd32c8b7211e445e3
+                        command:
+                          - "python3"
+                          - "/opt/k8s-init-container-nb-docs.py"
+                          - "/home/jovyan"
+                        volume_mounts:
```

```suggestion
                        volumeMounts:
```

These just get passed to k8s directly, so should be camelcase.